icu4c/source/i18n/unicode/listformatter.h

```diff
@@ -220,7 +220,6 @@ class U_I18N_API ListFormatter : public UObject{
     UnicodeString& format(const UnicodeString items[], int32_t n_items,
         UnicodeString& appendTo, UErrorCode& errorCode) const;
 
-#ifndef U_HIDE_DRAFT_API
```

Comment:
Keep this hidden for now

https://unicode-org.atlassian.net/browse/ICU-20809

Reply:
OK i see, the guard wasn't re-placed when it was un-promoted.

Reply:
fixed